packages/integrations/vercel/test/redirects.test.js

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+import { loadFixture } from './test-utils.js';
+
+describe('Redirects', () => {
+	/** @type {import('../../../astro/test/test-utils.js').Fixture} */
+	let fixture;
+
+	before(async () => {
+		fixture = await loadFixture({
+			root: './fixtures/redirects/',
+			redirects: {
+				'/one': '/',
+				'/two': '/',
+				'/blog/[...slug]': '/team/articles/[...slug]',
```

Comment:
What about object syntax?

Reply:
That is in a separate PR (added to the RFC after this was created): https://github.com/withastro/astro/tree/redirects-ssg-object